frontend/src/metabase/nav/components/WhatsNewNotification/utils.unit.spec.ts

```diff
@@ -44,6 +44,15 @@ describe("getLatestEligibleReleaseNotes", () => {
     ).toBe(undefined);
   });
 
+  it("should return nothing when white labeling, even if other conditions are satisfied", () => {
+    expect(
+      getLatestEligibleReleaseNotes({
+        ...DEFAULTS,
```

Comment:
We should probably update `DEFAULTS` with `isWhiteLabeling` as well.
TS would catch this for us if `getLatestEligibleReleaseNotes`'s parameters were not optional (as suggested in the other comment).